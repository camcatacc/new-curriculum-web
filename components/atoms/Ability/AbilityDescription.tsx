// Modules
import React from "react";

// Definitions
export interface AbilityDescriptionProps {
	children: React.ReactNode;
}

// Element
const AbilityDescription = ({ children }: AbilityDescriptionProps): JSX.Element => {
	return <>{children}</>;
};

export default AbilityDescription;
