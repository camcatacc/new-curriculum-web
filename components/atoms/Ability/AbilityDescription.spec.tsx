// Modules
import React from "react";
import { render } from "@testing-library/react";

// Elements
import AbilityDescription from "components/atoms/Ability/AbilityDescription";

// Tests
describe("AbilityDescription", () => {
	it("Renders as expected", () => {
		const { container } = render(<AbilityDescription>Hello, hello</AbilityDescription>);
		expect(container).toMatchSnapshot();
	});
});
