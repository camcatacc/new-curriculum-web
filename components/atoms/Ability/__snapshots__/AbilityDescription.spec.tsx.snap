// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AbilityDescription Renders as expected 1`] = `
<div>
  Hello, hello
</div>
`;
